sklearn/metrics/ranking.py

```diff
@@ -149,14 +149,34 @@ def average_precision_score(y_true, y_score, average="macro",
     sample_weight : array-like of shape = [n_samples], optional
         Sample weights.
 
+    interpolation : None (default), or 'eleven_point'
+
+        ``None``:
+            Do not interpolate the average precision. Instead, compute
+            ``sum(p[i] * (r[i] - r[i-1]))`` for all precision, recall pairs
+            ``p[i], r[i]`` for ``i >= 1``. This is the primary definition used
+            in the Wikipedia entry for Average precision. See References.
+        ``'eleven_point'``:
+            For each of the recall values, r, in {0, 0.1, 0.2, ..., 1.0},
+            compute the arithmetic mean of the first precision value with a
+            corresponding recall >= r. This is the metric used in the Pascal
+            Visual Objects Classes (VOC) Challenge and is as described in the
```

Comment:
Well, they describe it in the docs, though don't use it in the code. Maybe we should mention the docs.